src/Psy/Shell.php

```diff
@@ -279,7 +279,13 @@ public function run(InputInterface $input = null, OutputInterface $output = null
             $output = $this->config->getOutput();
         }
 
-        return parent::run($input, $output);
+        try {
+            return parent::run($input, $output);
+        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
+        }
+
+        $this->writeException($e);
```

Comment:
right, I've removed the cath for throwables, it's an other topic for an other PR